langmeta/langmeta/js/src/main/scala/org/langmeta/internal/io/PlatformFileIO.scala

```diff
@@ -33,7 +34,12 @@ object PlatformFileIO {
     result
   }
 
-  def write(path: AbsolutePath, proto: GeneratedMessage): Unit = {
+  def readAllDocuments(path: AbsolutePath): List[TextDocument] = JSIO.inNode {
+    val bytes = readAllBytes(path)
+    TextDocuments.parseFrom(bytes).documents
```

Comment:
Missing `.toList`, although I question why the signature is `List[T]` instead of `Seq[T]` which is what we have in the scalapb generated classes

Reply:
We only care about iterating over the list, so `Seq[T]` is perfectly fine for that IMO

Reply:
Will do.